docs/rules/order.md

```diff
@@ -89,3 +91,56 @@ You can set the options like this:
 ```js
 "import/order": ["error", {"groups": ["index", "sibling", "parent", "internal", "external", "builtin"]}]
 ```
+
+### `newlines-between: [bool]`:
+
+Checks existence of new lines between and inside import groups. For sake of examples let's assume `groups` settings like:
```

Comment:
I'm just proposing a new rewording here:
```
Enforces or forbids new lines between import groups.
If omitted, assertion messages will be neither enforced nor forbidden.
If set to true, a new line between each group will be enforced, and new lines inside a group will be forbidden.
If set to false, no new lines are allowed in the entire import section.
```

Btw, starting to think that having a string is more explicit (readable in the config), and more ESLint-like. `newlines-between: "always|never".